rise/static/main.js

```diff
@@ -275,6 +308,18 @@ function Revealer(selected_slide, config) {
   + require.toUrl("./reveal.js/css/reveal.css")
   + ' id="revealcss" />');
 
+  /* this policy of trying ./rise.css and then <notebook>.css
+   * should be redefinable in the config
+   */
+  if (window.location.pathname.endsWith('.ipynb')) {
+      // Attempt to load rise.css 
+      $('head').append(`<link rel="stylesheet" href="./rise.css" id="rise-custom-css" />`);
+      // Attempt to load CSS with the same path as the .ipynb but with .css extension instead
+      let notebook_css = window.location.pathname.replace(/\.ipynb$/,'.css');
+      $('head').append(`<link rel="stylesheet" href="${notebook_css}" id="notebook-custom-css" />`);
+
```

Comment:
Makes sense, you have a rise.css that could be more general applied first and then the notebook specific one applied on top of that...